packs/data/pfs-season-2-bestiary.db/disciple-of-urxehl-9-10.json

```
@@ -0,0 +1,3222 @@
+{
+    "_id": "U3RoZnfVAAKbrULX",
+    "data": {
+        "abilities": {
+            "cha": {
+                "mod": 7
+            },
+            "con": {
+                "mod": 3
+            },
+            "dex": {
+                "mod": 5
+            },
+            "int": {
+                "mod": 56
```

```suggestion
                "mod": 6
```
He is not that smart